ms2deepscore/BinnedSpectrum.py

```diff
@@ -0,0 +1,39 @@
+class BinnedSpectrum:
+    """Binned spectrum for use with MS2DeepScore."""
+    def __init__(self, binned_peaks: dict, metadata: dict):
+        self.peaks = binned_peaks
```

Comment:
It would be nice to document what binned_peaks looks like.

Reply:
Added a sentence to the docstring.